src/shared/schemas.ts

```diff
@@ -28,6 +28,7 @@ export interface SchemaMapping {
     path: string
```

Comment:
How much work would it be to make this URI instead of a file system path? Then we don't need `virtual`.